web-server/terraform.tfstate

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This file shouldn't be in Git - it holds current state which can be changed outside of the _Terraform_ scope.